sonar-project.properties

```diff
@@ -0,0 +1,17 @@
+sonar.projectKey=sap_cloud-commerce-spartacus-storefront
+
+sonar.projectName=Cloud commerce spartacus storefront
+sonar.projectVersion=0.1.0
+
+sonar.sources=projects/storefrontlib,projects/core,projects/storefrontstyles
+sonar.exclusions=**/node_modules/**,**/*.spec.ts
+
+sonar.tests=projects/storefrontlib,projects/core,projects/storefrontstyles,projects/storefrontapp-e2e
+sonar.test.inclusions=**/*.spec.ts
+
+sonar.typescript.lcov.reportPaths=coverage/lcov.info
```

Comment:
Coverage doesn't work in our setup, because artifacts aren't passed between jobs. For now I don't have a nice solution for it, but I will think about this problem.

Reply:
I think we also have a problem with path for coverage. Tests setup in our packages points to this one directory. So running tests for different packages we override them. We should create separate directory for each package.